plugins/pr-body-labels/src/index.ts

```diff
@@ -0,0 +1,26 @@
+import { Auto, IPlugin } from "@auto-it/core";
+
+/** Allow outside contributors to indicate what semver label should be applied to the Pull Request */
+export default class PrBodyLabelsPlugin implements IPlugin {
+  /** The name of the plugin */
+  name = "pr-body-labels";
+
+  /** Tap into auto plugin points. */
+  apply(auto: Auto) {
+    auto.hooks.prCheck.tapPromise(this.name, async ({ pr }) => {
+      if (!auto.git) {
+        return;
+      }
+
+      const projectLabels = await auto.git.getProjectLabels();
```

Comment:
Should this be limited to tags defined by the `.auto` config? Or does `getProjectLabels()` do that already? 
There's a potential of this being abused to add any label (i.e. "merge when built") to a PR simply by adding a note in the PR body.